wagtail/admin/forms/workflows.py

```diff
@@ -211,6 +211,8 @@ def get_workflow_edit_handler():
                 FieldPanel("task", widget=AdminTaskChooser(show_clear_link=False)),
             ],
             heading=_("Add tasks to your workflow"),
+            label=_("Task"),
+            icon="thumbtack",
```

Comment:
This (and the template changes to add icons) aren’t really needed, just a nice visual touch.